apps/src/lib/util/firehose.js

```diff
@@ -0,0 +1,114 @@
+/** @file Provides clients to AWS Firehose, whose data is imported into AWS Redshift. */
+
+import AWS from 'aws-sdk';
+
+const IDENTITY_POOL_ID = 'us-east-1:fbfec393-0afb-4682-84d4-59ad04a302f4'
+const AWS_REGION = 'us-east-1'
+
+AWS.config.region = AWS_REGION;
+AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+  IdentityPoolId: IDENTITY_POOL_ID,
+});
+const FIREHOSE = new AWS.Firehose({apiVersion: '2015-08-04'});
+
+/**
+ * A barebones client for posting data to an AWS Firehose stream.
+ * Usage:
+ *   FirehoseClient.putRecord(
+ *     'analysis-events',
+ *     {
+ *       created_at: DateTime.now,           // REQUIRED
```

Comment:
Looks like (based on my limited JS knowledge) the best route might be to use this [dateformat](https://www.npmjs.com/package/dateformat) NPM package?